Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -109,43 +117,73 @@ def generate_cpp(serialized_types):
     result.append('')
     result.append('#include "StreamConnectionEncoder.h"')
     for type in serialized_types:
-        for condition in type.conditions:
+        if type.condition is not None:
             result.append('')
-            result.append('#if ' + condition)
-        result.append('#include ' + type.header)
-        for condition in type.conditions:
-            result.append('#endif // ' + condition)
+            result.append('#if ' + type.condition)
+        for header in type.headers:
+            result.append('#include ' + header)
+        if type.condition is not None:
+            result.append('#endif // ' + type.condition)
     result.append('')
     result.append('namespace IPC {')
     for type in serialized_types:
         result.append('')
-        for condition in type.conditions:
-            result.append('#if ' + condition)
-        for encoder in ['Encoder', 'StreamConnectionEncoder']:
+        if type.condition is not None:
+            result.append('#if ' + type.condition)
+        for encoder in type.encoder_types():
             result.append('')
             result.append('void ArgumentCoder<' + type.namespace_and_name() + '>::encode(' + encoder + '& encoder, const ' + type.namespace_and_name() + '& instance)')
             result.append('{')
             for member in type.members:
-                result.append('    encoder << instance.' + member.name + ';')
+                if member.condition is not None:
+                    result.append('#if ' + member.condition)
+                if 'NULLABLE' in member.attributes:
```

Comment:
Personally, I find all uppercase a bit ugly and would have gone with "Nullable".

Reply:
I wanted it to stand out.  We can always adjust this as more attributes are added, and they will.